weblate/static/js/screenshots/clipboard-paste.js

```diff
@@ -0,0 +1,70 @@
+// Copyright © Michal Čihař <michal@weblate.org>
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+$(document).ready(() => {
+  // The paste trigger button
+  const $pasteScreenshotBtn = $("#paste-screenshot-btn");
+  // The file input to store the screenshot file
+  const $screenshotFileInput = $("#screenshot-form-container input#id_image");
+
+  $pasteScreenshotBtn.on("click", async (e) => {
+    e.preventDefault();
+    try {
+      // Read clipboard content
+      const clipboardItems = await navigator.clipboard.read();
```

Comment:
As I have slightly outdated Firefox, I ended up with:

```
TypeError: navigator.clipboard.read is not a function
```

It would be great to not show the button at all in such case.

Reply:
I see. Okay I will work on it.